plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/HeteroDescribableConfigurator.java

```diff
@@ -203,14 +203,14 @@ public CNode describe(T instance, ConfigurationContext context) {
 
     private Option<Descriptor<T>> lookupDescriptor(String symbol, CNode config) {
         return getDescriptors()
-            .filter(descriptor -> findByPreferredSymbol(descriptor, symbol) || findBySymbols(descriptor, symbol, config))
```

Comment:
Actually this is the correct formatting: https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/.idea/codeStyles/Project.xml#L89

😆

Reply:
Perhaps not, we should take the styling fix in a separate PR :)

Reply:
The format change was made automatically by IntelliJ. My vote is always to make format changes in separate PRs, but as your preference.